src/App.js

```diff
@@ -1,5 +1,108 @@
+const MissionUtils = require("@woowacourse/mission-utils");
+
 class App {
-  play() {}
+  constructor() {
+    this.computerNumberArray = this.getComputerRandomNumber();
+    this.userInputNumber;
+    this.countStrike = 0;
+    this.countBall = 0;
+    MissionUtils.Console.print('숫자 야구 게임을 시작합니다.');
+  }
+
+  getComputerRandomNumber() {
+    const computerNumberSet = new Set();
+    while (computerNumberSet.size < 3) {
+      computerNumberSet.add(MissionUtils.Random.pickNumberInRange(1,9));
+    }
+    const computerNumberArray = [...computerNumberSet];
+    return computerNumberArray;
+  }
+
+  play() {
+    MissionUtils.Console.readLine('숫자를 입력해주세요 :', (userInputNumber) => {
+      this.userInputNumber = userInputNumber;
+      this.isError();
+      this.count();
+      this.printMessage();
+      if (this.countStrike < 3) {
+        this.countStrike = 0;
+        this.countBall = 0;
+        this.play(); 
+      }
+      else if (this.countStrike === 3) {
+        MissionUtils.Console.print('3개의 숫자를 모두 맞히셨습니다! 게임 종료');
+        this.isGameRestart();
+      }
```

Comment:
```suggestion
      if (this.countStrike < 3) {
        this.countStrike = 0;
        this.countBall = 0;
        return this.play(); 
      }
      MissionUtils.Console.print('3개의 숫자를 모두 맞히셨습니다! 게임 종료');
      this.isGameRestart();
```
이런식의 표현은 어떨까요 ? if 문에 return을 해주면 else를 쓰지않아도 될것같습니다

Reply:
미처 생각 못했는데 코드가 더 깔끔해지네요!! 감사합니다ㅎㅎㅎ